packages/react/src/stories/ic-status-tag.stories.mdx

```diff
@@ -55,6 +55,16 @@ import readme from "../../../web-components/src/components/ic-status-tag/readme.
   <Story name="Small">
     <IcStatusTag label="Small" size="small" />
     <IcStatusTag label="Small" variant="outlined" size="small" />
+    <IcStatusTag label="Small" small="true" />
+  </Story>
```

Comment:
Please can you combine this into one 'Sizes story' - similar to this [ic-button size story](https://mi6.github.io/ic-ui-kit/branches/develop/web-components/?path=/story/web-components-button--size). This applies to the web components storybook as well